README.markdown

```diff
@@ -187,49 +233,76 @@ If Puppet is running on a 64-bit system, the 32-bit Registry key can be explicit
 
 For more on this parameter, see the [Purge existing values section](#purge-existing-values) under Usage.
 
+
 #### `registry_value`
 
-##### `path`
+Manages individual Registry values.
 
-*Optional.* Specifies a Registry value for Puppet to manage. Valid options: a string containing a Registry path. If any of the parent keys in the path don't exist, Puppet creates them automatically. For example: `HKLM\Software` or `HKEY_LOCAL_MACHINE\Software\Vendor`. Default value: the title of your declared resource.
+The title of the resource can determine the path and value name that will be managed, for example:
```

Comment:
Suggestion: omit can. "The title of the resource determines the..."

Reply:
Changed.  Also expanded on the note below that the path and value_name parameters will be used in preference to the resource title (implicit vs explicit settings)